src/main/java/com/cburch/logisim/util/LibraryUtil.java

```diff
@@ -56,6 +54,7 @@ public static String getName(Class cls) {
       }
     }
 
-    throw new NoSuchElementException("Missing " + ID_FIELD_NAME + " static const field for " + cls);
+    /* for backward compatibility we return the class name */
```

Comment:
Can we have oneliner comments using // FOO instead of /* FOO */?

Reply:
Done